pages/wishlist.html

```diff
@@ -67,125 +67,112 @@
+          <img class="card-img-vertical" src="/images/shallow-bowl.jpg" />
+          <span><i class="far fa-trash-alt card-delete-icon"></i></span>
+          <div class="card-details text-center">
+            <div class="title">Rounded shallow bowl</div>
             <div class="card-footer-vertical">
               <span class="discountedPrice">Rs. 499</span>
-              <span class="originalPrice">Rs. 999</span>
+              <span class="originalPrice pl-0p5">Rs. 999</span>
             </div>
           </div>
-          <button class="card-button">Move to Cart</button>
+          <button class="plain-button card-button mt-0p5">Add to Cart</button>
         </div>
 
-        <div class="card-vertical">
-          <img class="card-img-vertical" src="/images/cups.jpg" />
+        <div class="card-vertical m-1">
+          <img class="card-img-vertical" src="/images/white-plate1.jpg" />
           <span><i class="far fa-trash-alt card-delete-icon"></i></span>
-          <div class="card-details padding-sm">
-            <div class="title">Clay Craft</div>
+          <div class="card-details text-center">
+            <div class="title">White Ceramic Plate</div>
             <div class="card-footer-vertical">
               <span class="discountedPrice">Rs. 499</span>
-              <span class="originalPrice">Rs. 999</span>
+              <span class="originalPrice pl-0p5">Rs. 999</span>
             </div>
           </div>
-          <button class="card-button">Move to Cart</button>
+          <button class="plain-button card-button mt-0p5">Add to Cart</button>
         </div>
 
-        <div class="card-vertical">
-          <img class="card-img-vertical" src="/images/bottles.jpg" />
+        <div class="card-vertical m-1">
+          <img class="card-img-vertical" src="/images/cups.jpg" />
           <span><i class="far fa-trash-alt card-delete-icon"></i></span>
-          <div class="card-details padding-sm">
-            <div class="title">Clay Craft</div>
+          <div class="card-details text-center">
+            <div class="title">Set of Two Cups</div>
             <div class="card-footer-vertical">
               <span class="discountedPrice">Rs. 499</span>
-              <span class="originalPrice">Rs. 999</span>
+              <span class="originalPrice pl-0p5">Rs. 999</span>
             </div>
           </div>
-          <button class="card-button">Move to Cart</button>
+          <button class="plain-button card-button mt-0p5">Add to Cart</button>
         </div>
 
-        <div class="card-vertical">
+        <div class="card-vertical m-1">
           <div class="badge">
-            <img class="card-img-vertical" src="/images/cups.jpg" />
-            <span class="badge-text badge-on-left success">Bestseller</span>
+            <img class="card-img-vertical" src="/images/white-plate1.jpg" />
+            <span class="badge-text badge-on-left success px-0p5"
+              >Bestseller</span
```

Comment:
Formatting can be taken care of 👆🏽

Reply:
Prettier is formatting it. When I format and save, it goes back to this.